src/allotropy/parsers/moldev_softmax_pro/softmax_pro_structure.py

```diff
@@ -203,13 +203,15 @@ def create(reader: CsvReader) -> GroupData:
         )
 
         samples = data["Sample"].ffill()
-        return GroupData(
-            name=name,
-            sample_data=[
+        try:
+            sample_data = [
                 GroupSampleData.create(data.iloc[sample_entries.index])
                 for _, sample_entries in samples.groupby(samples)
-            ],
-        )
+            ]
+        except ValueError as e:
+            msg = "Invalid data - unsupported Group data format."
```

Comment:
Can we add a bit more detail here: 

```suggestion
            msg = f"Unable to read Group data format for group {name}."
```